bot.py

```diff
@@ -31,4 +31,41 @@ async def create_repo(ctx, repoName): #!git create_repo repo1
 	repo = user.create_repo(repoName)
 	await ctx.send("repository "+repoName+ " created!")
 
+
+# display open and closed issues
+
+@bot.command()
+async def issues(ctx, repoName, state): # !git issues MLH-Fellowship/github-discord-bot open
```

Comment:
Put `# !git issues MLH-Fellowship/github-discord-bot open` in the docstrings as well